src/LudoGame/GameEngine/Class1.cs

```diff
@@ -0,0 +1,90 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace GameEngine
```

Comment:
There is nothing wrong in you approach, but I would recommend to  only have one class per code file. So you should have one file called Game.cs containing the Game class, one called Player.cs containing the Player class, etc.

Reply:
This is now changed. All classes in GameEngine are now in separate files.